src/libs/page-detect.js

```diff
@@ -46,6 +46,8 @@ export const isCommit = () => isSingleCommit() || isPRCommit() || (isPRFiles() &
 
 export const isCompare = () => isRepo() && /^\/compare/.test(getRepoPath());
 
+export const isQuickPR = () => isCompare() && /[?&]quick_pull=1(&|$)/.test(location.search);
```

Comment:
It's funny that `?expand=1` has a different copy and no such branch link in the DOM.